### DISCUSSION TITLE
posts/docker-rancher-kubernetes/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#247] [Comments]
# posts/docker-rancher-kubernetes/

If you want to set up Kubernetes at home using Rancher to run Docker containers, this is the guide for you. This is a step by step tutorial of how to install and configure Rancher, Docker, and Kubernetes for your homelab. In this video we set up and configure a Minecraft server in just a matter of minutes with some enterprise like features. You can use this same process to spin up other Docker containers at home on your server or desktop.

https://technotim.live/posts/docker-rancher-kubernetes/

**Comment by jhmc93** [comment]
everytime i follow your video, the cluster just wont boot, no idea why it sayss [Disconnected] Cluster agent is not connected
